README_TH.md

```diff
@@ -57,26 +57,26 @@ PyThaiNLP provides standard NLP functions for Thai, for example part-of-speec ta
 - Command-line interface for basic functions, like tokenization and pos tagging (run `thainlp` in your shell)
 </details>
 
-Please see [our tutorials](https://www.thainlp.org/pythainlp/tutorials) on how to apply these functions to ML problems.
+อ่านรายละเอียดได้ที่ [tutorials](https://www.thainlp.org/pythainlp/tutorials)
 
 ## การติดตั้ง
 
 ```sh
 pip install --upgrade pythainlp
 ```
 
-This will install the latest stable release of PyThaiNLP.
-PyThaiNLP uses pip as its package manger and PyPI as its main distribution channel, see [https://pypi.org/project/pythainlp/](https://pypi.org/project/pythainlp/)
+วิธีดังขึ้นเป็นการติดตั้งรุ่นเสถียรของ PyThaiNLP
```

Comment:
`วิธีดังขึ้น` looks like a typo here, do you mean `วิธีดังกล่าว`?

Reply:
> 
> 
> `วิธีดังขึ้น` looks like a typo here, do you mean `วิธีดังกล่าว`?

Thank you.